### ISSUE TITLE
CS4 support

### ISSUE THREAD

**Issue by Rourke101** [fnordware/AdobeWebM#47]
Is it possible to get CS4 support? I have installed it but nothing happens and I cannot import webm files (it says the format is not recognised)

**Comment by fnordware** [comment]
No, sorry.  The plug-in is built for is CS5, the first version that was 64-bit.

If you knew how to do some programming you could maybe get it to build for CS4.

**Comment by 6081902811485155** [comment]
https://github.com/fnordware/AdobeWebM.git

**Comment by 6081902811485155** [comment]
<script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<script>
     (adsbygoogle = window.adsbygoogle || []).push({
          google_ad_client: "ca-pub-1012901709457135",
          enable_page_level_ads: true
     });
</script>

**Comment by 6081902811485155** [comment]
<script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>

<ins class="adsbygoogle"
     style="display:block"
     data-ad-client="ca-pub-1012901709457135"
     data-ad-slot="9822488359"
     data-ad-format="auto"
     data-full-width-responsive="true"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
